src/main/java/com/epam/test_generator/services/CaseService.java

```diff
@@ -127,6 +159,24 @@ public void removeCases(Long suitId, List<Long> caseIds) {
             });
     }
 
+    public void updateCases(long suitId, List<EditCaseDTO> editCaseDTOS) {
+        editCaseDTOS.forEach(caseDTO -> {
+            switch (caseDTO.getAction()) {
```

Comment:
If client provide null action than it will be null pointer exception. I recommend to set Action field in EditCaseDTO as notnull or to check on null value before switch operator.